src/main/java/edu/ksu/canvas/requestOptions/GetSubmissionsOptions.java

```diff
@@ -60,6 +60,16 @@ public GetSubmissionsOptions grouped(Boolean grouped) {
         addSingleItem("grouped", Boolean.toString(grouped));
         return this;
     }
+    
+    /**
+     * Filter the list of submissions by the given student ids..
+     * @param ids List of ids to filter submissions by
+     * @return This object to allow adding more options
+     */
+    public GetSubmissionsOptions studentIds(List<String> ids) {
+        optionsMap.put("student_ids[]", ids);
+        return this;
+    }
```

Comment:
Putting this in a class called `GetMultipleSubmissionsOptions` would make more sense as the two calls take very different arguments and normally the Options objects have constructors that take all the required arguments.

https://canvas.instructure.com/doc/api/submissions.html#method.submissions_api.index
https://canvas.instructure.com/doc/api/submissions.html#method.submissions_api.for_students